src/proposalBuilder/multiTxTransfer.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Box, Button, Divider, Flex, Icon } from '@chakra-ui/react';
+import { BiChevronDown, BiChevronUp } from 'react-icons/bi';
+import { v4 as uuid } from 'uuid';
+
+import { useAppModal } from '../hooks/useModals';
+import TextBox from '../components/TextBox';
+import { ParaMd } from '../components/typography';
+import { AsyncCardTransfer, PropCardError } from './proposalBriefPrimitives';
+
+const CONTRACT_ZERO =
+  '0x0000000000000000000000000000000000000000000000000000000000000000';
```

Comment:
ethers has the 0 address in it's constants

Reply:
Cool.